packages/transaction-controller/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [28.1.1]
+
+### Uncategorized
+
+- fix: simulation should handle NFT mints ([#4217](https://github.com/MetaMask/core/pull/4217))
+- Release/144.0.0 ([#4220](https://github.com/MetaMask/core/pull/4220))
```

Comment:
I moved this above to a **Changed** section, so if that suggestion looks good to you, you could remove this line:

``` suggestion
```